.github/workflows/ci.yml

```diff
@@ -24,7 +24,7 @@ jobs:
       run: pip install tox tox-gh-actions
 
     - name: Run Python tests
-      run: tox
+      run: tox -r
```

Comment:
Nit:
```suggestion
      make tests
```

Reply:
Does the GitHub workflow see `MAKEFILE`?

Reply:
Yes, it does see it. 
But we need to refine and decide better about the Makefile structure. In our dev docs, we share the tox in our CI using make in some parts. It's the second time we think about it ([first one](https://github.com/vmware/repository-service-tuf-cli/pull/110#pullrequestreview-1159892929)). What is good.😄 
I will merge and we can discuss it more in the below issue:
- https://github.com/vmware/repository-service-tuf/issues/229